setup.py

```diff
@@ -63,6 +64,49 @@ def clean_optional_plugins(self):
             os.chdir(root_dir)
 
 
+class SimpleCommand(Command):
```

Comment:
I had the same feeling, but setuptools vs distutils in python is confusing nowadays. And afaict, the recommendation is to adopt setuptools instead of distutils:

> Most Python users will not want to use this module directly, but instead use the cross-version tools maintained by the Python Packaging Authority. In particular, setuptools is an enhanced alternative to distutils that provides...

Reply:
> I had the same feeling, but setuptools vs distutils in python is confusing nowadays. And afaict, the recommendation is to adopt setuptools instead of distutils:
> 
> > Most Python users will not want to use this module directly, but instead use the cross-version tools maintained by the Python Packaging Authority. In particular, setuptools is an enhanced alternative to distutils that provides...
> 
> So, I'm trying to use distutils only when is necessary.

This is minor, but the fact that the `Command` class from setuptools practically comes from `distutils` make it more distutils then setuptools anyway IMO.

But this questioning actually came from the fact that I was checking what was really necessary on the `SimpleCommand` base class. For instance, I see no use for the `run_command` method, and the `setuptools.__init__()` already preserves the keyword arguments.

In effect, if I do:

```diff
diff --git a/setup.py b/setup.py
index 4a895b50f..8b34c4c9b 100755
--- a/setup.py
+++ b/setup.py
@@ -70,20 +70,10 @@ class SimpleCommand(Command):
 
     user_options = []
 
-    def __init__(self, *args, **kwargs):
-        """Store arguments so it's possible to call other commands later."""
-        super().__init__(*args, **kwargs)
-        self.__args = args
-        self.__kwargs = kwargs
-
     @abstractmethod
     def run(self):
         """Run when command is invoked."""
 
-    def run_command(self, command_class):
-        """Run another command with same __init__ arguments."""
-        command_class(*self.__args, **self.__kwargs).run()
-
     def initialize_options(self):
         """Set default values for options."""
 
```

I see no difference in behavior, everything works just the same.  Maybe I'm missing something, but I don't see the need for those two methods there.